magento/magento2
# Command to rebuild MySQL Triggers

### Description


### Fixed Issues (if relevant)
1. https://github.com/magento/magento2/issues/6478

### Manual testing scenarios
1. Start with a 2.0.x instance with Indexers set to By Schedule.
2. Upgrade to 2.1.8 or 2.1.9.
3. In the DB, attempt to delete a row from catalog_product_entity_varchar.
4. Observe error: `ERROR 1054: 1054: Unknown column 'entity_id' in 'NEW'`.
5. Run CLI command `./bin/magento indexer:rebuild-triggers`.
6. Retry step 3 with success this time.

### Contribution checklist
